sql/share/errmsg-utf8.txt

```diff
@@ -5131,1054 +5413,1351 @@ ER_NON_UPDATABLE_TABLE
         ger "Das Vermischen von Handlern in Partitionen ist in dieser Version von MariaDB nicht erlaubt"
         swe "Denna mix av lagringsmotorer är inte tillåten i denna version av MariaDB"
 ER_PARTITION_NOT_DEFINED_ERROR
+        chi "对于分区引擎，需要定义所有%-.64s"
         eng "For the partitioned engine it is necessary to define all %-.64s"
         ger "Für die partitionierte Engine müssen alle %-.64s definiert sein"
         swe "För partitioneringsmotorn så är det nödvändigt att definiera alla %-.64s"
 ER_TOO_MANY_PARTITIONS_ERROR
+        chi "定义了太多分区（包括子组分）"
         eng "Too many partitions (including subpartitions) were defined"
         ger "Es wurden zu vielen Partitionen (einschließlich Unterpartitionen) definiert"
         swe "För många partitioner (inkluderande subpartitioner) definierades"
 ER_SUBPARTITION_ERROR
+        chi "只有在子节分节的HASH/KEY分区中可以混合RANGE/LIST分区"
         eng "It is only possible to mix RANGE/LIST partitioning with HASH/KEY partitioning for subpartitioning"
         ger "RANGE/LIST-Partitionierung kann bei Unterpartitionen nur zusammen mit HASH/KEY-Partitionierung verwendet werden"
         swe "Det är endast möjligt att blanda RANGE/LIST partitionering med HASH/KEY partitionering för subpartitionering"
 ER_CANT_CREATE_HANDLER_FILE
+        chi "无法创建特定的处理程序文件"
         eng "Failed to create specific handler file"
         ger "Erzeugen einer spezifischen Handler-Datei fehlgeschlagen"
         swe "Misslyckades med att skapa specifik fil i lagringsmotor"
 ER_BLOB_FIELD_IN_PART_FUNC_ERROR
+        chi "分区功能中不允许BLOB字段"
         eng "A BLOB field is not allowed in partition function"
         ger "In der Partitionierungsfunktion sind BLOB-Spalten nicht erlaubt"
         swe "Ett BLOB-fält är inte tillåtet i partitioneringsfunktioner"
 ER_UNIQUE_KEY_NEED_ALL_FIELDS_IN_PF
+        chi "A%-.192s必须包含表的分区功能中的所有列"
         eng "A %-.192s must include all columns in the table's partitioning function"
 ER_NO_PARTS_ERROR
+        chi "不允许%-.64s = 0"
         eng "Number of %-.64s = 0 is not an allowed value"
         ger "Eine Anzahl von %-.64s = 0 ist kein erlaubter Wert"
         swe "Antal %-.64s = 0 är inte ett tillåten värde"
 ER_PARTITION_MGMT_ON_NONPARTITIONED
+        chi "不分区表上的分区管理是不可能的"
         eng "Partition management on a not partitioned table is not possible"
         ger "Partitionsverwaltung einer nicht partitionierten Tabelle ist nicht möglich"
         swe "Partitioneringskommando på en opartitionerad tabell är inte möjligt"
 ER_FOREIGN_KEY_ON_PARTITIONED
+        chi "不支持外键子句和分区结合使用"
         eng "Foreign key clause is not yet supported in conjunction with partitioning"
         ger "Fremdschlüssel-Beschränkungen sind im Zusammenhang mit Partitionierung nicht zulässig"
         swe "Foreign key klausul är inte ännu implementerad i kombination med partitionering"
 ER_DROP_PARTITION_NON_EXISTENT
+        chi "分区列表错误%-.64s"
         eng "Error in list of partitions to %-.64s"
```

Comment:
In d324c03d0c6395629213af98eaf5fc859d8eea12 this message was replaced with
```
ER_PARTITION_DOES_NOT_EXIST
        eng "Wrong partition name or partition list"
```
What would be the Chinese translation?

Reply:
This should work:
```
ER_PARTITION_DOES_NOT_EXIST
        chi "分区名称或分区列表错误"
        eng "Wrong partition name or partition list"
```

Reply:
Thank you! I see that @grooverdan already applied it.